onnxruntime/test/providers/nv_tensorrt_rtx/test_nv_trt_rtx_ep_util.cc

```diff
@@ -464,5 +472,266 @@ Ort::IoBinding generate_io_binding(
   return binding;
 }
 
+// Helper: make ONNX_NAMESPACE::TensorProto
+ONNX_NAMESPACE::TensorProto MakeTensor(const std::string& name,
+                                       ONNX_NAMESPACE::TensorProto::DataType dtype,
+                                       const std::vector<int64_t>& dims,
+                                       const std::vector<int64_t>& vals) {
+  ONNX_NAMESPACE::TensorProto t;
+  t.set_name(name);
+  t.set_data_type(dtype);
+  for (auto d : dims) t.add_dims(d);
+  for (auto v : vals) t.add_int64_data(v);
+  return t;
+}
+
+ONNX_NAMESPACE::TensorProto MakeTensorFloat(const std::string& name,
+                                            const std::vector<int64_t>& dims,
+                                            const std::vector<float>& vals) {
+  ONNX_NAMESPACE::TensorProto t;
+  t.set_name(name);
+  t.set_data_type(ONNX_NAMESPACE::TensorProto::FLOAT);
+  for (auto d : dims) t.add_dims(d);
+  for (auto v : vals) t.add_float_data(v);
+  return t;
+}
+
+OrtStatus* CreateModelWithNodeOutputNotUsed(const PathString& model_name) {
```

Comment:
for other tests that use models, we have checked in .onnx files and a script to generate them. is there a good reason to do it differently here? defining the model in a Python script (e.g., with onnxscript) could be more concise.